src/main/java/mServer/crawler/Crawler.java

```diff
@@ -19,6 +19,7 @@
  */
 package mServer.crawler;
 
+import java.nio.file.Paths;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.nio.file.Paths'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)